roles/dcos_cli/templates/promdash.json.j2

```diff
@@ -0,0 +1,39 @@
+{
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "network": "BRIDGE",
+      "image": "{{ promdash_image }}",
+      "portMappings": [{"containerPort": 3000, "hostPort": 0, "protocol": "tcp"}]
```

Comment:
"Marathon does not make use of these assigned service ports itself. It is the responsibility of a separate component to make use of this information and set up load balancing accordingly. A sample script that does this is provided with Marathon"

Set servicePort is not going to make any different for us as we are not considering it in our discovery solution at the moment as we are not discriminating by port at the load balancer level and we are relying on container-ip (weave)/container-port for internal dns communication.